js/utility.js

```diff
@@ -0,0 +1,39 @@
+/* utility */
+
+var utility = {
+  findContainer : function(elem, containerClass){
+      while ((elem = elem.parentElement) && !elem.classList.contains(containerClass));
+      return elem;
+  },
+  getProjectIdFromUrl: function(url) {
```

Comment:
Also... these are just quick url resolutions. In our next iteration these can be easily solved by external module / library.